<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Dialog::DlgCustomKeyboard</class>
 <widget class="QWidget" name="Gui::Dialog::DlgCustomKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabelCategory">
         <property name="text">
          <string>&amp;Category:</string>
         </property>
         <property name="buddy">
          <cstring>categoryBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="categoryBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="editCommand"/>
       </item>
       <item>
        <widget class="QTreeWidget" name="commandTreeWidget">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabelShortcut">
           <property name="text">
            <string>Current shortcut:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Gui::AccelLineEdit" name="accelLineEditShortcut">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabelNewShortcut">
           <property name="text">
            <string>Press &amp;new shortcut:</string>
           </property>
           <property name="buddy">
            <cstring>editShortcut</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Gui::AccelLineEdit" name="editShortcut"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Multi-key-shortcut delay: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Gui::PrefSpinBox" name="shortcutTimeout">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Time delay in miliseconds to disambiguate multi-key sequence shortcuts.
For example, pressing 'F' twice in less than the time delay setting here will
be treated as shorctcut key sequence 'F, F'.</string>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>100</number>
             </property>
             <property name="value">
              <number>300</number>
             </property>
             <property name="prefPath" stdset="0">
              <cstring>Shortcut/Settings</cstring>
             </property>
             <property name="prefEntry" stdset="0">
              <cstring>ShortcutTimeout</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="textLabelAssigned">
           <property name="toolTip">
            <string>This list shows commands having the same shortcut in the priority from hight
to low. If more than one command with the same shortcut are active at the
same time. The one with the highest prioirty will be triggered.</string>
           </property>
           <property name="text">
            <string>Shorcut priority list:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="assignedTreeWidget">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>0</height>
            </size>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="buttonAssign">
           <property name="text">
            <string>&amp;Assign</string>
           </property>
           <property name="shortcut">
            <string>Alt+A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonClear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonReset">
           <property name="text">
            <string>&amp;Reset</string>
           </property>
           <property name="shortcut">
            <string>Alt+R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonResetAll">
           <property name="text">
            <string>Re&amp;set All</string>
           </property>
           <property name="shortcut">
            <string>Alt+S</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>41</width>
             <height>150</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDown">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Down</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="texLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabelDescription">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Gui::AccelLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Gui/Widgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefSpinBox</class>
   <extends>QSpinBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>categoryBox</tabstop>
  <tabstop>editCommand</tabstop>
  <tabstop>commandTreeWidget</tabstop>
  <tabstop>accelLineEditShortcut</tabstop>
  <tabstop>editShortcut</tabstop>
  <tabstop>assignedTreeWidget</tabstop>
  <tabstop>buttonAssign</tabstop>
  <tabstop>buttonClear</tabstop>
  <tabstop>buttonReset</tabstop>
  <tabstop>buttonResetAll</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
